_posts/2.5/2021-06-09-new.md

```diff
@@ -67,7 +67,39 @@ Private guest lobby message, display position in waiting line
 
 A new API endpoint was added: `/insertDocument`. Now, it is possible to send a batch of documents with all the common presentation parameters such as `current`, `downloadable` and `removable`. 
 
-All the presentations are sent via payload (the body of the request), whether by a link to download the file or by `base64` encoded file.
+All the presentations are sent via payload (the body of the request) such as when preuploading documents in `/create` endpoint, whether by a link to download the file or by `base64` encoded file.
```

Comment:
Consider just linking to the api page section insertDocument, so you don't need to copy the same info twice